docs/createFlows.rst

```diff
@@ -80,6 +79,26 @@ Azkaban supports many job types. You just need to specify it in ``type``, and ot
 
 You need to write your own pig script and put it in your project zip and then specify the path for the pig.script in the config section.
 
+Common Parameters
+########
+Besides the type and the dependencies parameters, there are several parameters that Azkaban reserves for all jobs. All of the parameters below are optional.
+
++----------------+----------------------------------------------------------------------------------------------------------------------------------+
+| Parameter      | Description                                                                                                                      |
++================+==================================================================================================================================+
+| retries        | The number of retries that will be automatically attempted for failed jobs                                                       |
+| retry.backoff  | The millisec time between each retry attempt                                                                                     |
+| working.dir    | Override the working directory for the execution. This is by default the directory that contains the job file that is being run. |
```

Comment:
plese keep them consistent, either put . at the end of all of them or not having them for all.